src/lib/webpack/run-webpack.js

```diff
@@ -30,13 +33,15 @@ const devBuild = async (env, onprogress) => {
 
 				let protocol = devServer.https ? 'https' : 'http';
 				let host = process.env.HOST || devServer.host || 'localhost';
-				let port = process.env.PORT || devServer.port || 8080;
 
-				let serverAddr = `${protocol}://${host}:${port}`;
-				let localIpAddr = `${protocol}://${ip.address()}:${port}`;
+				let serverAddr = `${protocol}://${host}:${chalk.bold(port)}`;
+				let localIpAddr = `${protocol}://${ip.address()}:${chalk.bold(port)}`;
 
-				process.stdout.write(chalk.green('Compiled successfully!!\n\n'));
-				process.stdout.write('You can view the application in the browser.\n\n');
+				process.stdout.write(chalk.green('\nCompiled successfully!!\n\n'));
+				if (parseInt(port, 10) !== parseInt(config.devServer.port, 10)) {
```

Comment:
Showing this message only when it is not using default port. So, it will be easy for the users to understand what port it has been changed to !!